variants/Cargo.lock

```diff
@@ -630,13 +630,6 @@ dependencies = [
  "util-linux",
 ]
 
-[[package]]
-name = "libinih"
```

Comment:
nit: some cleanup in this commit.

Reply:
Yeah, I'll do this cleanup if I can, I've tried twice in rebases to fix it but I'll take another shot at it.